src/utils/crypto.ts

```diff
@@ -8,7 +8,7 @@ type Algorithm = {
   alias: string
 }
 
-type Data = string | boolean | number | object | ArrayBufferView | ArrayBuffer | ReadableStream
+type Data = string | boolean | number | object | ArrayBufferView | ArrayBuffer
```

Comment:
```suggestion
type Data = string | boolean | number | object
```

Although not directly related to this PR, I think we can remove `ArrayBuffer`, `ArrayBufferView`, etc. since they are subtypes of `object`.